contrib/azure/azure-user-data-template

```diff
@@ -0,0 +1,25 @@
+#cloud-config
+coreos:
+  units:
+    - name: format-ephemeral.service
```

Comment:
The cloud-config is parsed at boot, so this will wipe the filesystem every time the host is rebooted. You'll need to add a check to see if the filesystem has been created first. See https://github.com/deis/deis/blob/master/contrib/gce/gce-user-data-template#L9,L15